app/views/works/homepage.html.erb

```diff
@@ -0,0 +1,81 @@
+<div>
+  <section class="spotlight">
+    <h3>
+      <span class="media-spotlight">Media Splotlight:
+      </span>
+      <%= link_to(@works.show_spotlight.title, work_path(@works.show_spotlight)) %>
+      by Steely Dan
+    </h3>
+    <p>12 Votes - Greate American rock band</p>
+  </section>
+  <hr class="root-hr">
+</div>
+
+<div class="grid-x">
+  <div class="cell small-4">
+    <strong>
+      <h3>
+        Top Movies
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?